### DISCUSSION TITLE
How do I stop using custom domain for my github page?

### DISCUSSION THREAD

**Discussion by ForkKILLET** [community/community#23188] [Other Feature Feedback, Questions, & Ideas]
<p>I tried to just clear the input box but failed.<br>
Thanks.</p>

**Comment by ForkKILLET** [comment]
<p>It said <code>No changes to custom domain.</code></p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Do you have a CNAME file in your repository? If you do, you’ll need to delete it.</p>

**Comment by ForkKILLET** [comment]
<p>Thanks <a class="mention" href="/canuckjacq">@canuckjacq</a><br>
Deleted, but still cannot leave the custom domain blank.</p>

**Comment by canuckjacq** [comment]
<p>It looks as though you had two branches deployed. I’ve fixed that now so only the <code>delta-refactor</code> branch is deployed. Give it an hour and check back, just so everything has a chance to work.</p>

**Comment by canuckjacq** [comment]
<p>Could you try changing your publishing source to the master branch?</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source" target="_blank" rel="noopener">Configuring a publishing source for your GitHub Pages site - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source" target="_blank" rel="noopener">//docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by ForkKILLET** [comment]
<p>But it has already been deployed on master branch 40m ago.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/3/13f529a540ef1a6a8056eec418b0d8f10adac5df.jpeg" data-download-href="/uploads/short-url/2QykuE6z22gEdCnEM1zN102Qu0D.jpeg?dl=1" title="Screenshot_20210126_222030_com.quark.browser" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/35412022/181075521-dd5f98d8-9b2a-4976-9779-8521ff1454ce.jpeg" alt="Screenshot_20210126_222030_com.quark.browser" data-base62-sha1="2QykuE6z22gEdCnEM1zN102Qu0D" width="230" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/3/13f529a540ef1a6a8056eec418b0d8f10adac5df_2_230x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/3/13f529a540ef1a6a8056eec418b0d8f10adac5df_2_345x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/3/13f529a540ef1a6a8056eec418b0d8f10adac5df_2_460x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/3/13f529a540ef1a6a8056eec418b0d8f10adac5df_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_20210126_222030_com.quark.browser</span><span class="informations">1080×2340 212 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by canuckjacq** [comment]
<p>Which content are you expecting to see when you load your site? (is it the <code>index.html</code>, for example?)</p>

**Comment by ForkKILLET** [comment]
<p>Yes, the index.html in master branch.</p>

**Comment by canuckjacq** [comment]
<p>Can you try loading your site now?</p>

**Comment by ForkKILLET** [comment]
<p>In fact I’m working on <code>ForkFG/HardWayNazo</code>, and its <a href="https://forkfg.github.io/HardWayNazo/" rel="noopener nofollow ugc">page</a> is correctly deployed.<br>
When I visit <a href="https://forkfg.github.io" rel="noopener nofollow ugc">https://forkfg.github.io</a>, I’m redirected to my custom domain.</p>

**Comment by ForkKILLET** [comment]
<p>Oh it works now. Thanks very much!</p>

**Comment by canuckjacq** [comment]
<p>That’s fantastic. There was something a bit glitchy going on in the background, but it should behave for you now.</p>
